docs/source/DeveloperDocumentation/TaskEngineWorkflowFeatures.md

```diff
@@ -391,3 +391,71 @@ The AVOD example below shows three clips being stitched together. The first clip
   }
 }
 ```
+
+```json
+
+```
+
+## LIVE COMPOSE WITH MANIFEST MANIPULATION
+
+The Vualto Task Engine now supports manifest manipulation to generate LIVE COMPOSE streams (VOD playlist looping), using AWS Mediatailor Channel Assembly. This workflow takes VOD streaming URLs directly and the resulting Live stream fragments come frome the original VOD streaming URLs - this could result in cost savings in caching layers in some instances.
```

Comment:
> this could result in cost savings in caching layers in some instances.

Arguably you only need to say one of these two phrases; "_could result in_" or "_in some instances_". There's technically nothing wrong with both, but you probably don't need both.